src/components/SignIn.vue

```diff
@@ -1,24 +1,23 @@
 <template>
 	<div>
-		<div ref='test' @click="startSignIn">Sign In</div>
+		<button @click="signInHandler">Sign In</button>
 		<span v-if="error">{{error}}</span>
 	</div>
 	
 </template>
 
 <script>
+import { signIn } from '../services/auth'
+
 export default {
 	data: () => ({
 		error: null
 	}),
-	mounted() {
-		gapi.signin2.render(this.$refs.test, {
```

Comment:
This method doesn't seem to be very friendly with Vue, keeps throwing a style error on router change, so you'll have to style the signin button yourself.